Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -857,7 +857,7 @@ namespace Multiplayer
             EnableAutonomousControl(controlledEntity, AzNetworking::InvalidConnectionId);
         }
         
-        AZLOG_INFO("Multiplayer operating in %s mode", GetEnumString(m_agentType));
+        AZLOG_INFO("Multiplayer operating in %s mode\n", GetEnumString(m_agentType));
```

Comment:
Isn't a new line added by default to messages? Is the guidance from sig-core that all such messages need to have a new line at the end? Note, that a new line can be different between different platforms.

Reply:
This won't add a newline to the actual .log file. I'm adding /n so that stdout has a newline. without the newline, the processcommunicator reading of the server's output will be one giant run-on-sentence :P
I originally automatically would have added a newline to the fwrite, but it was recommended against.

Reply:
AZ::ILogger will inject the \n on your behalf for the AZ::Trace subcall, but leave it off for the actual IConsole log so that it still displays nicely in the console (otherwise you end up with double-spaced log output in the console).

Reply:
AZ::ILogger corresponds to all the AZLOG_(...) macro calls for those who are wondering.

Reply:
Oh, will me adding /n affect IConsole then?  I've checked the log files, and the in-game console... both look good. For example, here's this line "Multiplayer operating in %s mode/n"
![image](https://user-images.githubusercontent.com/32776221/144320900-be598e93-a7a6-4da0-b1fc-e152798a91e2.png)
![image](https://user-images.githubusercontent.com/32776221/144320944-9876528e-91b8-4e8e-8345-248f6c56730a.png)

Reply:
I'd verify that it doesn't affect IConsole in code.

Reply:
Before that I need to figure out what iConsole does besides manage cvars >.<

Reply:
As far I can see AZLOG flows into the normal AZ::Debug::Trace.  Looking at iConsole seems like it handles cvars, but not responsible for printing logs.  Can anyone point me towards what might be getting too many newlines?

Reply:
Ahh I see... LoggerSystemComponent.cpp does this... 
        // Force a new-line before calling the AZ::Debug::Trace functions, as they assume a newline is present
        buffer[length + 1] = '\n';
In a string, the buffer[length] is the null-terminator. We're added a character after the null-terminator... meaning this line does nothing. 
Check out my update to LoggerSystemComponent.cpp here. I just removed the confusing lines which were actually doing nothing beneficial to the string :)